apps/web/src/services/cost-explorer/store/cost-query-set-store.ts

```diff
@@ -30,9 +31,19 @@ export const useCostQuerySetStore = defineStore('cost-query-set', {
         },
     },
     actions: {
+        async listDataSources(): Promise<void> {
+            try {
+                const { results } = await SpaceConnector.clientV2.costAnalysis.dataSource.list();
+                this.dataSourceList = results;
+            } catch (e) {
+                ErrorHandler.handleError(e);
+                this.dataSourceList = [];
+            }
+        },
         async listCostQuerySets(): Promise<void> {
             try {
                 const { results } = await SpaceConnector.client.costAnalysis.costQuerySet.list({
```

Comment:
todo: this should be changed to clientV2

Reply:
Oh I know.. But not working now so I'll fix it. Anyway I'll add annotation now!